Cargo.toml

```diff
@@ -105,7 +107,7 @@ rand_distr = { version = "0.5", optional = true }
 tracing = { version = "0.1.40", optional = true }
 
 atomic_float = { version = "1.1.0", optional = true }
-rtrb = "0.3.2"
+rtrb = { version = "0.3.2", optional = true }
```

Comment:
`inquire` as `dev-dependency` is also optional right?

Reply:
dev-dependencies are not allowed to be optional